src/coreclr/gc/gcinterface.dac.h

```diff
@@ -169,6 +175,11 @@ struct opaque_gc_heap
     uint8_t unused;
 };
 
+struct opaque_generation
+{
+    uint8_t unused;
+};
```

Comment:
nit - instead of naming these `opaque`, is it better to name them `unused`? and can you please add a comment to explain why they are here?